modules/data.atmosphere/R/met.process.R

```diff
@@ -55,35 +55,32 @@ met.process <- function(site, input_met, start_date, end_date, model, host, dbpa
           
           args <- list(outfolder, start_date, end_date)
           cmdFcn  = paste0(pkg,"::",fcn,"(",paste0("'",args,"'",collapse=","),")")
-          remote.execute.R(cmdFcn,host$name,user=NA, verbose=TRUE)
-          
-          mimetype = 'application/x-netcdf'
-          formatname = "NARR"
-          check <- dbfile.input.check(site.id, start_date, end_date, mimetype, formatname, con=con, hostname=fqdn())
+          new.files <- remote.execute.R(cmdFcn,host$name,user=NA, verbose=TRUE)
           
+          print("start CHECK")
+          check = dbfile.input.check(site$id, start_date, end_date, 
+                                     mimetype=new.files$mimetype[1], formatname=new.files$formatname[1], 
+                                     con=con, hostname=new.files$host[1])
+          print("end CHECK")
+          print(check)
           if(length(check)>0){
-            raw.id = check$container_id[1]
+            raw.id <- list(input.id=check$container_id, dbfile.id=check$id)
           }else{
-            raw.id <- dbfile.input.insert(in.path = outfolder, 
-                                          in.prefix = "NARR", 
-                                          siteid = 1135, 
+            raw.id <- dbfile.input.insert(in.path=dirname(new.files$file[1]),
+                                          in.prefix=new.files$dbfile.name[1], 
+                                          siteid = site$id,  
                                           startdate = start_date, 
                                           enddate = end_date, 
-                                          mimetype =  mimetype, 
-                                          formatname = formatname,
+                                          mimetype=new.files$mimetype[1], 
+                                          formatname=new.files$formatname[1],
                                           parentid = NA,
                                           con = con,
-                                          hostname = host$name)$input.id
-            raw.id <- newinput$input.id #1000000127
+                                          hostname = host$name)
           }
     }else{
-      raw.id <- 1000000127  
+      raw.id <- list(input.id = 1000000127, dbfile.id = NA) 
```

Comment:
I tried to explain that earlier in the comments in line 49 - the raw NARR met was downloaded, but ultimately deleted to save space because it isn't needed anymore. So instead here I was hardcoding in the database input.id so it wouldn't re-download it. It's clunky and I'm literally in the process of fixing it with the generalization fix using registration xml files that Mike and I came up with.